aten/src/ATen/native/Gesv.cpp

```diff
@@ -72,6 +78,111 @@ static void applyGesv(Tensor& b, Tensor& A, std::vector<int64_t> infos) {
   }
 }
 
+std::tuple<Tensor&,Tensor&> _gesv_single_out_cpu(
+    Tensor& sol, Tensor& lu,
+    const Tensor& self, const Tensor& A) {
+#ifndef USE_LAPACK
+  AT_ERROR("gesv: LAPACK library not found in compilation");
```

Comment:
Ah okay I know why the build's failing. We had a hacky `skipIfLapack` rule that checks the error message but this isn't fitting the pattern. LOL

Can you make this throw `gesv : Lapack library not found in compile time` instead? Would be great to change the one above too. Thanks! :) I'll add a proper flag for LAPACK later.